pkg/batchscheduler/volcano/volcano_scheduler.go

```diff
@@ -90,6 +95,10 @@ func (v *VolcanoBatchScheduler) syncPodGroupInClusterMode(app *v1beta2.SparkAppl
 	if _, ok := app.Spec.Driver.Annotations[v1beta1.KubeGroupNameAnnotationKey]; !ok {
 		//Both driver and executor resource will be considered.
 		totalResource := sumResourceList([]corev1.ResourceList{getExecutorRequestResource(app), getDriverRequestResource(app)})
```

Comment:
No need to calculate the totalResource if customRequestResource is configured. It's better move totalResource back a little bit.

Reply:
good advice. if customRequestResource is configured, it's no need to calculate getExecutorRequestResource again. You mean the same with @hzxuzhonghu 's  last advice